frontend/package-lock.json

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
I think if we use yarn instead of npm in the package.json stuff this should be in yarn-lock instead of package-lock